docs/execution-providers/MIGraphX-ExecutionProvider.md

```diff
@@ -84,8 +85,60 @@ You can check [here](https://github.com/scxiao/ort_test/tree/master/python/run_o
+
+Items are added as a python dictionary when invoking the migraphx execution provider
+
+|Session Option Flag | Values | Description|
+|---|---|---|
+| device_id | INT | Selecit the device ID specified for the session run (default will be device 0) | 
+| migraphx_fp16_enable | 1 or 0 | Enable FP16 quantization mode via the MIGraphX API of the input model|
+| migraphx_int8_enable | 1 or 0 | Enable int8 static quantization mode of the input model via the MIGraphX API.\n Requires calibration table path vars to be set.|
+| migraphx_int8_calibration_table_name | <absolute path to calibration table> | Path to a set of input calibration data for int8 static model quantization |
+| migraphx_int8_use_native_calibration_table | 1 or 0 | Use a calibration table from Nvidia native int8 format or json dumped format|
+| migraphx_save_compiled_model | 1 or 0 | Enable saving a model as an MIGraphX (.mxr) format after compile |
+| migraphx_save_compiled_path  | <string> - Path to write .mxr file (default is ./compiled_model.mxr) | Path where the MIGraphX compiled model is stored. |
+| migraphx_load_compiled_model | 1 or 0 | Enable loading a model as an MIGraphX (.mxr) format after compile |
+| migraphx_load_compiled_path | <string> - Path to read .mxr file (default is ./compiled_model.mxr) | Path where the MIGraphX compiled model is stored. |
+| migraphx_exhaustive_tune | 1 or 0 (default 0) | Enable exhaustive tuning of parameters as part of compilation via the MIGraphX API. Adds additional compile time for a potential perf boost.|
+| migraphx_mem_limit | INT | Set the memory limit used for memory arena. Default uses ORTs default_memory_arena_cfg value. |
+| migraphx_external_alloc | Address | Address of external memory allocator function used for this EP. Useful for reading in larger models weights. |
+| migraphx_external_free | Address  | Address of external memory deallocator function used for this EP. Useful for unloadng what was allocated with the migraphx_external_alloc input |
+| migraphx_external_empty_cache | Address  | Address of external memory cache used for this model. Useful for caching results of externally allocated models. |
+
+
+## Environment Variaables
+
+Envirinoment variables can be invoked on a global level. These are typically used via 
+
+```
+export ORT_MIGRAPHX_XXXXX = <value> 
+```
+
+or invoked before running a binary as
+
+```
+ORT_MIGRAPHX_XXXX=<value> python3 example_script.py
+```
+
+This will start an inference session and superceed inputs invoked via Session() 
+
+|Environment Option Flag | Values | Description|
+|---|---|---|
+| ORT_MIGRAPHX_DUMP_MODEL_OPS | 1 or 0 | Enable dumping of model operators during parsing | 
+| ORT_MIGRAPHX_FP16_ENABLE | 1 or 0 | Enable FP16 quantization mode via the MIGraphX API of the input model|
+| ORT_MIGRAPHX_INT8_ENABLE | 1 or 0 | Enable int8 static quantization mode of the input model via the MIGraphX API.\n Requires calibration table path vars to be set.|
```

Comment:
Can you point to the specific variables that need to be set? If it's below, can you add something like "(see below)"?
```suggestion
| ORT_MIGRAPHX_INT8_ENABLE | 1 or 0 | Enable int8 static quantization mode of the input model via the MIGraphX API.\n Requires calibration table path vars to be set. |
```